exporting/prometheus/remote_write/remote_write.c

```diff
@@ -40,17 +31,9 @@ int prometheus_remote_write_send_header(int *sock, struct instance *instance)
         "Content-Type: application/x-www-form-urlencoded\r\n\r\n",
         connector_specific_config->remote_write_path,
         instance->config.destination,
-        buffer_strlen((BUFFER *)instance->buffer));
-
-    size_t header_len = buffer_strlen(header);
-    ssize_t written = send(*sock, buffer_tostring(header), header_len, flags);
+        buffer_strlen(simple_connector_data->last_buffer->buffer));
 
-    buffer_flush(header);
-
-    if (written != -1 && (size_t)written == header_len)
-        return 0;
-    else
-        return 1;
+    return;
```

Comment:
This `return` line is always implict added when the compiler has a `void` function on `amd64` or `x86` computers, but in normal situation we are not adding this for our `void` functions, as you can see [here](https://github.com/netdata/netdata/blob/2e2a855935fce6ad9025e81ed479bdbc883d197d/daemon/main.c#L128)  and [here](https://github.com/netdata/netdata/blob/2e2a855935fce6ad9025e81ed479bdbc883d197d/daemon/main.c#L102), so I will suggest you remove this.

Reply:
Removed.